Extend/Thank_You_Controller.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+use \Wirecard\Oxid\Core\Helper;
+use \OxidEsales\Eshop\Core\Registry;
+
+/**
+ * Class Thank_You_Controller
+ *
+ * @package Wirecard\Extend
+ *
+ */
+class Thank_You_Controller extends Thank_You_Controller_parent
+{
+    /**
+     * Extends the parent init method and deletes a wdtoken
+     */
+    public function init()
+    {
+        Registry::getSession()->deleteVariable("wdtoken");
+        parent::init();
+    }
+
+    /**
+     * @return bool
+     */
+    public function getOrderStatus()
```

Comment:
Could we use better name? Confusing when you see that method in template and expect it returns order status, but it returns bool.